packages/storage/amplify_storage_s3_dart/lib/src/model/s3_get_url_plugin_options.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/amplify_core.dart';
+
+part 's3_get_url_plugin_options.g.dart';
+
+/// {@template storage.amplify_storage_s3.get_url_plugin_options}
+/// The configurable parameters invoking the Storage S3 plugin `getUrl`
+/// API.
+/// {@endtemplate}
+@zAmplifySerializable
+class S3GetUrlPluginOptions extends StorageGetUrlPluginOptions {
+  /// {@macro storage.amplify_storage_s3.get_url_plugin_options}
+  const S3GetUrlPluginOptions({
+    Duration expiresIn = const Duration(minutes: 15),
```

Comment:
Why is this duration 15min vs. 1hour in the other ctors?

Reply:
good catch, seems like an existing code issue and I think it should be 15 min everywhere. @HuiSF please confirm and I can fix other ctors to 15 min

Reply:
Ahh my bad, missed updating the other ctor, the default duration is 15 mins. Thanks for catching this and fixing.